domain/src/test/java/uk/gov/hmcts/ccd/definition/store/domain/validation/casefield/CaseFieldEntityACLValidatorImplTest.java

```diff
@@ -29,7 +29,7 @@
     private CaseFieldEntity caseField;
 
     @Mock
-    private UserRoleEntity userRole;
+    private AccessProfileEntity userRole;
```

Comment:
rename variable

Reply:
Fixed in https://github.com/hmcts/ccd-definition-store-api/pull/961/commits/3951e2bdda580027372daf452a54fbcde899194f